README.md

```diff
@@ -144,8 +144,36 @@ The following field applies to multiple types of web mapping services:
 
 | Field Name   | Type      | Description |
 | ------------ | --------- | ----------- |
+| roles        | \[string] | A list of roles, see the list below. | 
 | href:servers | \[string] | A list of replacement values for `{s}` in `href`s. |
 
+#### roles
+
+Several roles can be assigned that provide an indicator for the web map library how to handle the link.
+
+**One of** the following roles may bet set to describe the type of layer. 
+If no role is provided, `data` is assumed to be the default.
+
+| Role    | Description |
+| ------- | ----------- |
+| basemap | A basemap. |
+| overlay | An overlay with transparent background to show on top of basemaps. |
+| data    | A layer with the actual data to show on top of the other layer types. |
```

Comment:
Could you provide an example of a "layer sandwich" including base, overlay and data? In general, I have the feeling that in some cases the overlay could be on top of the data layer. A basic example is Google Maps with its data layers (Public transit, Traffic etc.) which shows between the background (Default, Satellite or Terrain) and the overlay (street/POI names).
Or would you consider the e.g. traffic data as base layer on top of the other base layer and no data layer? 🤔